magento/inventory
# Add AdminGws support to MSI.

### Description (*)


### Fixed Issues (if relevant)

1. none.

### Manual testing scenarios (*)

1. Create additional website.
2. Create additional stock and source for new website.
3. Create product with default source and custom assigned.
4. Create admin user with restriction for main website.
5. Navigate to source grid and verify default source is not visible.
6. Navigate to stock grid and verify default stock is not visible.
7. Navigate to product grid and verify only custom source and stock visible in product row.
8. Open product edit page and verify only custom source and stock visible in product row.
9. Add some changes to custom source and save product.
10. Login with admin user without restrictions.
11. Navigate to product edit page and verify only custom source has changed.


### Questions or comments


### Contribution checklist (*)
